Example/Tests/JSZVCRRecorderTestCase.bundle/testRecordingLargeChunkedNetworkCall.plist

```diff
@@ -0,0 +1,1065 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<array>
+	<dict>
+		<key>cancelled</key>
+		<false/>
+		<key>data</key>
+		<dict>
+			<key>data</key>
+			<data>
+			KioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioq
```

Comment:
This is an encoded version of 30000 bytes of `*` characters. We check the length explicitly in one of the tests.